### DISCUSSION TITLE
huTool真的是很好的一个组件了

### DISCUSSION THREAD

**Discussion by TrReMix** [looly/hutool#2290] [General]
在项目里 可能除了Json模块我会用Jackson
其他的我基本上都会用huTool来搞定了

**Comment by looly** [comment]
欢迎及时反馈问题。

Hutool的JSON模块是根据org.json改造的，性能稍差一点，但是使用上还是比较简单的（应该比Jackson简单。）

**Reply by githubzhangfei** [reply]
用jsonutil 的时候会把List<Map>转成空数据。我没有证据，但是结果告诉我的是有这种情况

**Comment by chaoweimt** [comment]
@claude 用jsonutil 的时候会把List转成空数据。我没有证据，但是结果告诉我的是有这种情况
